src/many-client/src/client/ledger.rs

```diff
@@ -0,0 +1,63 @@
+use std::collections::BTreeMap;
+
+use many_identity::{Address, CoseKeyIdentity};
+use many_modules::ledger::{BalanceArgs, BalanceReturns, InfoReturns, SendArgs};
+use many_protocol::ManyError;
+pub use many_types::ledger::{Symbol, TokenAmount};
+use many_types::VecOrSingle;
+
+use crate::ManyClient;
+
+#[derive(Clone, Debug)]
+pub struct LedgerClient {
+    client: ManyClient,
+}
+
+impl LedgerClient {
+    pub fn new(client: ManyClient) -> Self {
+        LedgerClient { client }
+    }
+
+    pub async fn symbols(&self) -> Result<BTreeMap<String, Symbol>, ManyError> {
```

Comment:
The functions exposed by this structure should match the specification, what do you think? So `info()`, `balance()`, `send()`, ...

Reply:
I agree, but I'm thinking of adding more methods as I write specification tests for them.

Reply:
@hansl I thought more about it and I've matched those, I'll adapt the specification tests to match